assets/airports/kdca.json

```diff
@@ -571,80 +571,108 @@
           "draw": [["ADAXE", "BEBLE", "COVTO", "FERGI", "MELOE", "HORTO"], ["CAPVC", "GAITE", "LBOLT", "RLLLL", "HORTO"], ["HORTO", "BUFFR*"], ["HORTO", "JERES*"]]
       }
   },
-  "departures": {
-    "airlines": [
-        ["awi", 12],
-        ["eny", 10],
-        ["aal", 9],
-        ["flg", 9],
-        ["rpa", 8],
-        ["asq", 8],
-        ["jbu", 6],
-        ["vrd", 4],
-        ["jia", 4],
-        ["ual", 3],
-        ["swa", 3],
-        ["fft", 2],
-        ["vrd", 2]
-    ],
-    "destinations": [
-        "REBLL3", "HORTO2"
-    ],
-    "type": "random",
-    "offset": 0,
-    "frequency": 37.5
-  },
-  "arrivals": [
-    {
-      "_name": "CAPSS TWO SOUTH",
-      "altitude": 16000,
-      "type": "cyclic",
-      "frequency": 17.5,
-      "period": 75,
-      "offset": 25,
-      "speed": 280,
-      "fixes": ["BULII", "CAPSS", "FLIPZ", "MOEJO", "JILES", "SMOOT", "BAAAM", "PACKE"],
-      "airlines": [
-          ["awi", 9],
-          ["eny", 7],
-          ["flg", 6],
-          ["rpa", 6],
-          ["asq", 4],
-          ["aal", 4],
-          ["jbu", 3],
-          ["swa", 3],
-          ["vrd", 3],
-          ["jia", 3],
-          ["ual", 2],
-          ["fft", 2],
-          ["vrd", 2]
-      ]
+  "spawnPatterns": [
+      {
+          "category": "departure",
+          "origin": "kdca",
+          "destination": "",
+          "speed": "",
+          "altitude": "",
+          "method": "random",
+          "offset": 0,
+          "rate": 18.75,
+          "airlines": [
+            ["awi", 12],
+            ["eny", 10],
+            ["aal", 9],
+            ["flg", 9],
+            ["rpa", 8],
+            ["asq", 8]
+          ],
+        "route": "KDCA.REBLL3.OTTO"
```

Comment:
This should be `OTTTO`. Not sure if its a mistake only here, or elsewhere in the file. But it's definitely supposed to have three T's (in fact, I worked a few airplanes filed over that fix today).